apps/app/src/routes/_layout/profile/_tabs/my-feeds.tsx

```
@@ -0,0 +1,172 @@
+              : "You haven't created any feeds yet."}
+          </p>
+          {!filters.term && filters.value === "all" && (
+            <Button variant="filled" onClick={handleCreateNewFeed}>
+              Create Your First Feed
+            </Button>
+          )}
+        </div>
+      ) : (
+        <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4 sm:gap-5">
+          {filteredFeeds &&
+            filteredFeeds.map((feed) => <FeedCard key={feed.id} feed={feed} />)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)